source/docs/pipeline_and_webpack.md

```diff
@@ -0,0 +1,29 @@
+title: Asset Pipeline & Webpack
+---
+
+### Specifying a mode
+
+Embark has by default two modes for the building assets:
+
+* `development` - Is faster to build, produces sourcemaps, but also outputs bigger js filesizes. Is the default mode of `embark run` and `embark console`
+* `production` - Is slower to build but also outputs much smaller js filesizes. Is the default mode of `embark build` and `embark upload`
+
+You can also specify a particular mode with `--pipeline`, for e.g
+
+<pre><code class="shell">$ embark run --pipeline production</code></pre>
+
+### Default webpack config
+
+Embark has a internal default webpack config, this config allows you out of the box to:
+
+* ES6 modules & imports
+* Importing CSS, SCSS & Fonts in JS
+* Support for React & JSX
+
```

Comment:
Could add...

+ Automatic support for older browsers via babel's `[preset-env](https://babeljs.io/docs/en/babel-preset-env)`. Embark's default [Browserlist](https://github.com/browserslist/browserslist) setting is `['last 1 version', 'not dead', '> 0.2%']`.